force-app/main/default/classes/StorageChargeCheckTest.cls

```diff
@@ -0,0 +1,53 @@
+@isTest
+public with sharing class StorageChargeCheckTest 
```

Comment:
test is good, just please change formatting and make it private and remove `with sharing` keyword